resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.service.erb

```diff
@@ -10,7 +10,12 @@
 #
 [Unit]
 Description=<%= EZBake::Config[:project] %> Service
-After=syslog.target network.target
+
+<% if EZBake::Config[:build_type] == 'foss' -%>
```

Comment:
hm, maybe. I'm not really sure where we could put this to load it in for ezbake o_o